File: clase9/InnerClass/PYTHON/OuterClass.py
class OuterClass:
    #atributos
    class InnerClass:
        #atributos
        #metodos
        def __init__(self):
            print("Crea un objeto de la clase InnerClass\n")
            self.y = 5

    #metodos
    def __init__(self):
        print("Crea un objeto de la clase OuterClass\n")
        self.x = 10

File: RAI/clase9/InnerClass/JAVA/OuterClass.java
public class OuterClass{
    //Atributos
    int x = 10;
    //InnerClass
    class InnerClass{
        //atributos
        int y = 5;
        //Metodos
        public InnerClass(){
            System.out.println("Se crea la Clase InnerClass");
        }

    }
    //Metodos
    public OuterClass(){
        System.out.println("Se crea la Clase OuterClass");
    }
}